integration-tests/actions/automationv2/actions.go

```diff
@@ -517,7 +573,48 @@ func (a *AutomationTest) SetConfigOnRegistry() error {
 			ocrConfig.TypedOnchainConfig21 = a.RegistrySettings.Create21OnchainConfig(a.Registrar.Address(), a.UpkeepPrivilegeManager)
 		} else if a.RegistrySettings.RegistryVersion == ethereum.RegistryVersion_2_2 {
 			ocrConfig.TypedOnchainConfig22 = a.RegistrySettings.Create22OnchainConfig(a.Registrar.Address(), a.UpkeepPrivilegeManager, a.Registry.ChainModuleAddress(), a.Registry.ReorgProtectionEnabled())
+		} else if a.RegistrySettings.RegistryVersion == ethereum.RegistryVersion_2_3 {
+			ocrConfig.TypedOnchainConfig23 = a.RegistrySettings.Create23OnchainConfig(a.Registrar.Address(), a.UpkeepPrivilegeManager, a.Registry.ChainModuleAddress(), a.Registry.ReorgProtectionEnabled())
+			ocrConfig.BillingTokens = []common.Address{
+				common.HexToAddress(a.LinkToken.Address()),
+			}
+
+			ocrConfig.BillingConfigs = []i_automation_registry_master_wrapper_2_3.AutomationRegistryBase23BillingConfig{
+				{
+					GasFeePPB:         100,
+					FlatFeeMilliCents: big.NewInt(500),
+					PriceFeed:         common.HexToAddress(a.EthUSDFeed.Address()),
+					Decimals:          18,
+					FallbackPrice:     big.NewInt(1000),
+					MinSpend:          big.NewInt(200),
+				},
+			}
 		}
+		// TODO the SetConfigTypeSafe has different signature in v23
+
+		//function setConfigTypeSafe(
+		//	address[] memory signers,
+		//	address[] memory transmitters,
+		//	uint8 f,
+		//	OnchainConfig memory onchainConfig, // No this
+		//	uint64 offchainConfigVersion,
+		//	bytes memory offchainConfig,
+		//	IERC20[] memory billingTokens, // not this
+		//	BillingConfig[] memory billingConfigs // not this
+		//)
+
+		//ocrConfig := contracts.OCRv2Config{
+		//	Signers:               signers,
+		//	Transmitters:          transmitters,
+		//	F:                     f,
+		//	OffchainConfigVersion: offchainConfigVersion,
+		//	OffchainConfig:        offchainConfig,
+		//}
+
+		// TODO log here to see what is ocrConfig
+		a.Logger.Info().Msgf("=====ocrConfig.BillingTokens: %v", ocrConfig.BillingTokens)
+		a.Logger.Info().Msgf("=====ocrConfig.BillingConfigs: %v", ocrConfig.BillingConfigs)
+		a.Logger.Info().Msgf("=====ocrConfig: %v", ocrConfig)
 		err = a.Registry.SetConfigTypeSafe(ocrConfig)
```

Comment:
This is throwing:

```
5:18PM WRN No error data in tx
5:18PM WRN Failed to decode revert reason
5:18PM WRN Custom revert reason is supported by pragma, but we could not decode it. This might be a bug in Seth. Please contact the Test Tooling team. Pragma=0.8.19
5:18PM WRN Failed to parse value error="strconv.ParseInt: parsing \"\": invalid syntax" Value=

```